src/pages/CustomerInvoicePage.js

```diff
@@ -390,7 +406,12 @@ export const CustomerInvoicePage = ({
     });
   };
 
-  const onSearchChange = () => {};
+  const onSearchChange = searchTerm => {
+    dispatch(filterData(searchTerm));
```

Comment:
would be cleaner and make the func given to onChange have this better variable name if you just put debounce in here?

Reply:
I think I should do a PR for `SearchBar` to accept an optional `action` prop so we can bypass this function altogether. I haven't tried but I *think* putting `debounce` in here will return a new function each call, resulting in all invocations not having the same reference to the timeout variable to cancel consecutive calls?